docker/Dockerfile-prod

```diff
@@ -2,61 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM python:3.8-alpine AS assets
+FROM python:3.9-slim AS assets
 
-COPY requirements.txt /app/requirements.txt
-COPY docker/*.sh /usr/local/bin
+COPY requirements.txt /requirements.txt
+COPY system_requirements.txt /system_requirements.txt
 
-# Install pure-Python, compiled, and OS package dependencies.  Use scanelf to
-# uninstall any compile-time OS package dependencies and keep only the run-time
-# OS package dependencies.
-RUN install-deps.sh /app/requirements.txt
+RUN \
+    apt-get update && \
+    xargs apt-get install -y < /system_requirements.txt && \
+    apt-get clean
+
+RUN pip install -r /requirements.txt
 
 COPY . /app
 RUN pip install --no-cache /app
-
-ARG CIRCLE_SHA1
-ARG CIRCLE_BUILD_URL
-RUN echo "{ \
-        \"commit\": \"$CIRCLE_SHA1\", \
-        \"version\": \"$CIRCLE_SHA1\", \
-        \"source\": \"https://github.com/mozilla-conduit/lando-ui\", \
-        \"build\": \"$CIRCLE_BUILD_URL\" \
-    } \
-    " > /version.json
-
-RUN apk add --update ruby && rm -rf /var/cache/apk/*
 RUN gem install sass --no-document -v 3.4.25
 
+# TODO get rid of the below.
```

Comment:
Is this work for the future, or for this PR?

Reply:
This is future work. the `auth` module needs to be refactored a little before these can be removed.